### ISSUE TITLE
[shared_preferences] Failed to get value for key when app re launch

### ISSUE THREAD

**Issue by bbrto21** [flutter-tizen/plugins#165]
~I don't know why, but at some point this plugin was broken~

**How to reproduce**
precondition : Tizen 6.5 
1. launch example app
2. touch + button a few times.
3. terminate example app
4. re launch

**Comment by swift-kim** [comment]
Any logs or error messages?

**Comment by bbrto21** [comment]
What I know now is preference cache only has a boolean type value when re-launched :cry:

**Comment by bbrto21** [comment]
Oddly enough, when initializing the cache table, it succeeded for a boolean type.

https://github.com/flutter-tizen/plugins/blob/a643927aad218b375a76d5b2c3821a342ef0d86c/packages/shared_preferences/lib/shared_preferences_tizen.dart#L30

**Comment by bbrto21** [comment]
not only boolean, get value APIs for each type that bound with native actually returns 0(success) 
hmm... I think this is a native API bug... I found this recently refactored.

 ```bash
D/CAPI_APPFW_PREFERENCE(16681): stub.cc: preference_set_int(121) > [SECURE_LOG] key(flutter.counter), value(32)
D/CAPI_APPFW_PREFERENCE(16826): stub.cc: preference_get_int(184) > [SECURE_LOG] key(flutter.counter), value(32)
D/CAPI_APPFW_PREFERENCE(17524): stub.cc: preference_get_boolean(198) > [SECURE_LOG] key(flutter.counter), value(32)
D/CAPI_APPFW_PREFERENCE(18944): stub.cc: preference_get_boolean(198) > [SECURE_LOG] key(flutter.counter), value(32)

 ```

**Comment by bbrto21** [comment]
As a result of checking, this is a phenomenon that appears in Tizen 6.5 or higher.

**Comment by bbrto21** [comment]
I reported this to the person in charge of Application Preferences API.
He said it will be fixed in the next release.

**Comment by bbrto21** [comment]
It resolved in Tizen 6.5 latest release
